### DISCUSSION TITLE
开源协议更新：现已允许再次分发衍生作品

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#5217] [公告]
如果您基于该存储库创作衍生作品，或合理使用了本存储库中的较大量内容，您必须：
1. 给出适当的署名，且提供指向本存储库的链接。GitHub 上的 Fork 可视为满足此条。
2. 不得以任何方式暗示开发者为您或您的使用作担保。
3. 给出适当的说明，以明确表示该衍生作品是非官方内容。
4. 使用本指南文件作为协议，不得附加额外条款。
5. 该衍生作品的名称必须以 “PCL” 或 “Plain Craft Launcher” 开头，并在其后添加后缀。例如 “PCL 整合包优化版”、“PCL PR Collection”。
6. 该衍生作品不得包含 PCL 的赞助解锁功能，或与其用途相近的功能。包括但不限于：主题切换（自定义软件颜色/皮肤）、回声洞投稿等。
7. 以善意目的进行合理使用，不要用 PCL 来干坏事、恰烂钱、整烂活。开发者可能会将撤销所有授权作为最终手段。

查看完整协议：https://github.com/Hex-Dragon/PCL2/blob/main/LICENCE

**Comment by LinQingYuu** [comment]
>现已允许再次分发衍生作品

PCL 自定义主页魔改版（

**Comment by LinQingYuu** [comment]
>现已允许再次分发衍生作品

衍生作品是否持有再许可的权利（指发布衍生作品的魔改版）

**Reply by LTCatt** [reply]
衍生作品必须使用相同协议，所以自然可以 :D

**Comment by Pigeon0v0** [comment]
> 6. 该衍生作品不得包含 PCL 的赞助解锁功能，或与其用途相近的功能。包括但不限于：主题切换（自定义软件颜色/皮肤）、回声洞投稿等。

打包主题包，但是不包括自定义软件颜色属于违反这一条吗

或者可否具体展开这一条到特定功能？

**Reply by Silverteal** [reply]
详细解释：PCL-Community此前计划将其PCL-CE的主题替换为绿色，以与原版区分，但不确定这是否违反许可协议

**Reply by LinQingYuu** [reply]
> 详细解释：PCL-Community此前计划将其PCL-CE的主题替换为绿色，以与原版区分，但不确定这是否违反许可协议

我认为应该是违反的（不确定龙猫是不是这么想的，但至少我认为是这样的

**Reply by LTCatt** [reply]
是违反的，我修改一下来去除歧义……抱歉（
`该衍生作品不得包含 PCL 的赞助解锁功能，或与其实质相近的功能或内容。包括但不限于：主题（例如修改软件颜色/皮肤）、回声洞等。`

**Reply by Pigeon0v0** [reply]
我还是有两点需要确认一下...

第一个还是打包主题包，我原意是指不包括快照版才能解锁的内容

第二个是全局性的更改主题颜色，比如接下来的版本大家都变成绿色

**Reply by WForst-Breeze** [reply]
> 第一个还是打包主题包，我原意是指不包括快照版才能解锁的内容

该内容属于切换主题，应当可以归类于 `PCL 的赞助解锁功能，或与其实质相近的功能或内容`，即使衍生版本所设计的主题不是 PCL 原版的，只要是允许「切换主题」这一用户行为的就是不允许的。

> 第二个是全局性的更改主题颜色，比如接下来的版本大家都变成绿色

改内容不属于用户行为，个人认为应该不违反该条目。

**Reply by LTCatt** [reply]
umm 我觉得全局的主题颜色改变本身就属于 “赞助解锁功能”，因为即使不能切换，在旁人看来也可能和快照版的主题混淆。
例如，如果我把整个 PCL 的配色都改成橙色， **即使不允许自由切换，但在旁人看来也和 “我解锁了活跃橙!” 是差不多的……**

改成了 `该衍生作品不得包含 PCL 的赞助解锁功能，或与其相近的功能或内容。例如将配色修改为蓝色色系以外的颜色。`
希望能够理解 Orz

**Reply by Deep-Dark-Forest** [reply]
> 改成了 `该衍生作品不得包含 PCL 的赞助解锁功能，或与其相近的功能或内容。例如将配色修改为蓝色色系以外的颜色。`

那我把颜色改成很浅的蓝可以吗？

**Reply by LTCatt** [reply]
这个可以，蓝色系就行

**Reply by Hill23333** [reply]
是否能制作深色模式？（依然是蓝色色系，只不过白色部分变成黑色）

**Reply by LTCatt** [reply]
可以（

**Reply by Deep-Dark-Forest** [reply]
能否添加切换深蓝，浅蓝和龙猫蓝的选项？

**Reply by LTCatt** [reply]
可以

**Comment by MoYuan-CN** [comment]
参考 PCL 的部分设计方式，但是完全无代码重合的情况下，是否属于衍生作品？
或者说，在这种情况下，是否需要保证相同许可证，抑或是可以使用双许可证/其他开源许可证？

**Reply by LTCatt** [reply]
umm 是打算做成怎样的一个形式？能简单说下么（

**Reply by Hill23333** [reply]
借鉴PCL的界面，但是完全重写（应该是这个意思）

**Reply by Silverteal** [reply]
都没说到重点，实际上是用**另一种语言**参考PCL的逻辑和页面重写

题外话：把它叫做PCL我一直是反对的

**Reply by MoYuan-CN** [reply]
> 实际上是用另一种语言参考PCL的逻辑和页面重写

对的，仅仅是参考 PCL 的逻辑和页面设计，实际代码是完全不同的语言

**Reply by wuliaodexiaoluo** [reply]
代码应该可以用其他协议，但是署名应该是不可避免的

**Reply by LTCatt** [reply]
可以按照少量使用来处理 =。=
